identity-controller/src/app/models/database/database.model.ts

```diff
@@ -0,0 +1,140 @@
+import * as mongo from 'mongodb';
+import { IInvitationRecord } from '../interfaces/invitation-record';
+import { TypedEvent } from '../typed-event/typed-event.model';
+
+export interface IDBEvent {
+  operation: string;
+  _id?: string;
+  ref: string;
+}
+
+export type DatabaseNameType = 'identity';
```

Comment:
The database name should be pulled-in from the environment variables, as it may not be the same in the prod environment. There should already be a `DB_NAME` environment variable that is being passed-in.

I don't seem to see this variable used elsewhere though, so we may even be able to nuke it?

Reply:
can be nuked